tests/quota/awsNodeSdk.js

```diff
@@ -252,7 +257,7 @@ function multiObjectDelete(bucket, keys, callback) {
     }, callback);
 }
 
-describe('quota evaluation with scuba metrics', function t() {
+runIfMongo('quota evaluation with scuba metrics', function t() {
```

Comment:
as it's not used anywhere else : 
`(process.env.S3METADATA === 'mongodb' ? describe : describe.skip)`